docs/en_US/NAS/CDARTS.md

```diff
@@ -0,0 +1,34 @@
+# CDARTS
+
+## Introduction
+CDARTS builds a cyclic feedback mechanism between the search and evaluation networks. First, the search network generates an initial topology for evaluation, so that the weights of the evaluation network can be optimized. Second, the architecture topology in the search network is further optimized by the label supervision in classification, as well as the regularization from the evaluation network through feature distillation. Repeating the above cycle results in a joint optimization of the search and evaluation networks, and thus enables the evolution of the topology to fit the final evaluation network.
+

```

Comment:
This pr looks great to me. Only one comment: please introduce cdarts trainer and cdarts mutators here. For example, parameters of cdarts trainer ([here](https://github.com/microsoft/nni/blob/master/docs/en_US/NAS/DARTS.md) is an example of darts, [this](https://nni.readthedocs.io/en/latest/NAS/DARTS.html) is its rendering), the difference of the cdarts mutators.